docs/index.md

```diff
@@ -0,0 +1,26 @@
+# GCM Documentation Index
```

Comment:
If think we may want to rename this file to `README.md` so that when you navigate to the `/docs` directory in the web UI you also see this page:

<img width="1132" alt="image" src="https://user-images.githubusercontent.com/5658207/199520874-b06afc31-3740-4ef0-bdb6-b15ee6164215.png">